zulip/zulip/__init__.py

```diff
@@ -989,12 +989,12 @@ def get_profile(self, request=None):
             request=request,
         )
 
-    def get_presence(self, email):
+    def get_user_presence(self, email):
         # type: (Dict[str, Any]) -> Dict[str, Any]
         '''
             Example usage:
 
-            >>> client.get_presence()
+            >>> client.get_user_presence()
```

Comment:
doesn't this need an argument in the example?

Reply:
Added the argument, and also renamed the example file.